internal/provider/tests_resource_test.go

```diff
@@ -81,17 +80,17 @@ resource "imagetest_tests" "foo" {
 		},
 		// TODO: This test will leave the clusters dangling and needs to be
 		// manually cleaned up.
-		"k3sindocker-enters-debug-mode-with-proper-exit-code-on-failure": {
-			{
-				Config:      fmt.Sprintf(k3sindockerTpl, "k3s-in-docker-fails-with-proper-exit-code.sh"),
-				ExpectError: regexp.MustCompile(`.*213.*`),
-				PreConfig: func() {
-					if err := os.Setenv("IMAGETEST_SKIP_TEARDOWN", "true"); err != nil {
-						t.Fatal(err)
-					}
-				},
-			},
-		},
+		// "k3sindocker-enters-debug-mode-with-proper-exit-code-on-failure": {
```

Comment:
Could we add a comment explaining why we keep it here ?

Reply:
the comment is right above as a TODO :D

I'll toggle it back on once I can figure out the right terraform test framework incantation to set test scoped env vars